MekHQ/build.gradle

```diff
@@ -140,26 +142,26 @@ task compileMML (type: GradleBuild) {
 }
 
 task mmlJar (type: Jar, dependsOn: compileMML) {
-    archiveName = "MegaMekLab.jar"
+    archiveFileName = "MegaMekLab.jar"
     from "${mmlDir}/build/classes/java/main"
     from "${mmlDir}/build/resources/main"
     manifest {
         attributes "Main-Class": 'megameklab.com.MegaMekLab'
-        attributes 'Class-Path' : "${mmJar.archiveName} " + (project.sourceSets.main.runtimeClasspath.files
+        attributes 'Class-Path' : "${mmJar.archiveFileName} " + (project.sourceSets.main.runtimeClasspath.files
             .findAll { it.name.endsWith(".jar") && !it.name.toLowerCase().startsWith("megamek")}
             .collect { "${lib}/${it.name}" }.join(' '))
     }
-    ext.jarLocation = "${buildDir}/libs/${archiveName}"
+    ext.jarLocation = "${buildDir}/libs/${archiveFileName}"
     inputs.dir "${mmlDir}/build/classes/java/main"
```

Comment:
And here.

Reply:
Same here. The MML build directory is in the top level.